util/chunk/chunk_test.go

```diff
@@ -614,45 +643,62 @@ func (s *testChunkSuite) TestSwapColumn(c *check.C) {
 	chk1.AppendFloat32(0, 1)
 	chk1.MakeRef(0, 1)
 	chk1.AppendFloat32(2, 3)
+	row1 := chk1.GetRow(0)
 
 	// chk2: column1 refers to column0
 	chk2 := NewChunkWithCapacity(fieldTypes, 1)
-	chk2.AppendFloat32(0, 1)
+	chk2.AppendFloat32(0, 12)
```

Comment:
semantic changed?

Reply:
Yes. Before it only tests "checkRef", and the values don't matter actually. 
I want to check that the values are swapped, so I changed the values and added some assertions.